atlas/populate_atlas.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/expect
+
+set username [lindex $argv 0];
+set password [lindex $argv 1];
+
+spawn /etc/atlas/bin/quick_start.py
```

Comment:
change /etc/atlas to /usr/lib/atlas/apache-atlas-1.2.0/

Reply:
Done.